ansible/configs/sap-hana/sample_vars_ec2.yml

```diff
@@ -0,0 +1,35 @@
+cloud_provider: ec2
+env_type: sap-hana
+aws_region: us-east-1
+HostedZoneId: Z09717862MUAYG7P3BER6
+email: mak@redhat.com
+software_to_deploy: none
+aws_access_key_id: XXXXXXXXXXXX
+aws_secret_access_key: XXXXXXXXXXXX
```

Comment:
those should come from secrets

I would not put secrets here in a versioned directory, but rather include the secret when deploying, like:

```
ansible-playbook main.yaml -e @configs/sap../sample_vars.yml  -e @~/mysecrets/sandbox1234.yml
```